test/test_language.rb

```diff
@@ -385,7 +385,8 @@ def test_all_languages_have_a_language_id_set
   end
 
   def test_all_languages_have_a_valid_id
-    invalid = Language.all.select { |language| language.language_id < 0 || language.language_id >= (2**31 - 1) }
+    deleted_language_ids = [21]  # Prevent re-use of deleted language IDs
```

Comment:
21 was removed in https://github.com/github/linguist/pull/3933. This prevents re-use "to be safe".